ccd-definition/CaseEventToFields/DrawDirectionsOrder-SDO-nonprod.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "Draw_Directions_Order_Judge",
+    "CaseFieldID": "DirectionOrderClaimTrackRequired",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "ClaimsTrack",
+    "PageDisplayOrder": 1,
```

Comment:
I think from designs these changes are for the second page, so should this be 2?